src/components/Filter.js

```diff
@@ -0,0 +1,75 @@
+// import * as json from "src/mock-data/tickets.json";
+import {useState} from "react";
+
+export function Filter() {
+    const [all, setAll] = useState(true);
+    const [noStops, setNoStops] = useState(false);
+    const [oneStop, setOneStop] = useState(false);
```

Comment:
Чтобы не дублировать ```useState()```, можно заменить на единый useState с числом остановок
```suggestion
const [stops, setStops] = useState(0); // от 0 до 3
```